### PR TITLE
fix invalid filter in GetProp make storage crashed

### PR DESCRIPTION
**Pull request by critical27**
## What type of PR is this?
- [X] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 

https://github.com/vesoft-inc/nebula-ent/issues/1165
#4565 

#### Description:

Invalid filter will make storage crash, this only happens in enterprise version with additional filter push down rule.

## Checklist:
Tests:
- [X] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by wenhaocs — APPROVED** [review]
LGTM

**Review by panda-sheep — APPROVED** [review]

**Review by SuperYoko — APPROVED** [review]

File: src/storage/exec/QueryUtils.h [hunk 1]
```diff
@@ -73,6 +73,9 @@ class QueryUtils final {
 
       if (nullType == NullType::UNKNOWN_PROP) {
         VLOG(1) << "Fail to read prop " << propName;
+        if (!field) {
```
Comment (czpmango) [inline src/storage/exec/QueryUtils.h:76]: Why not just return null outside the if block？

Comment (critical27) [reply src/storage/exec/QueryUtils.h:76]: It is null here~ NullType::UNKNOWN_PROP

Comment (czpmango) [reply src/storage/exec/QueryUtils.h:76]: IMO, NullType::UNKNOWN_PROP is not necessary, which can lead to more problems due to graphd's unreasonable handling of runtime errors.
For example, filter operator execution will throw a runtime error if UNKNOWN_PROP is fed to it. 
https://github.com/vesoft-inc/nebula/blob/7f185c969f33ac7bc85a57de0ec3d7620253c24b/src/graph/executor/query/FilterExecutor.cpp#L67-L69

**Pull request merged**